tools/spades_header_fixer/spades_header_fixer.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="spades_header_fixer" name="SPAdes fasta header fixer" version="1.1.1">
+	<description>Fixes Spades Fasta ids</description>
+	<command >sed -r "s/>NODE(_[0-9]+)_(.*)/>${input.name}\1 \2/g" $input >$output</command>
+	<inputs>
+		<param name="input" type="data" format="fasta" label="fix these fasta headers"/>
+		</inputs>
+	<outputs>
+		<data name="output" format="fasta"/>
+	</outputs>
+  <tests>
+    <test>
+         <param name="input" value="spades_before.fasta" />
+         <output name="output" file="spades_header_fixed.fasta" ftype="fasta"/>
+    </test>
+  </tests>
+
+	<help>
+	
+	</help>
+</tool>
```

Comment:
Add citations section like [wade.xml](https://github.com/phac-nml/galaxy_tools/blob/master/tools/wade/wade.xml#L166) . Put author as me.

Reply:
Added in commit 2946ebc